integreat_cms/cms/templates/pages/page_form_sidebar/settings_box.html

```diff
@@ -84,12 +84,16 @@
                 <label>
                     {% translate "Additional permissions for this page" %}
                 </label>
-                <p class="italic">
-                    {% translate "This affects only users, who don't have the permission to change arbitrary pages anyway." %}
-                </p>
-                <div id="page_permission_table">
-                    {% include "pages/_page_permission_table.html" %}
-                </div>
+                {% if page_id %}
+                    <p class="italic">
+                        {% translate "This affects only users, who don't have the permission to change arbitrary pages anyway." %}
```

Comment:
```suggestion
                        {% translate "This only affects users who do not have permission to edit arbitrary pages." %}
```
I know this is not really part of this PR, but I'd suggest somewhat changing this formulation. The current formulation is slightly confusing because it implies that *no* user has page editing permissions.

Reply:
Thank you! It took us a while to settle on a suitable German translation, but now we found something we're satisfied with :smile: